src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.ts

```diff
@@ -59,11 +51,98 @@ export class IscsiTargetDetailsComponent implements OnChanges, OnInit {
     }
   };
 
+  private detailTable: TableComponent;
+  @ViewChild('detailTable')
+  set content(content: TableComponent) {
+    this.detailTable = content;
+    if (content) {
+      content.updateColumns();
+    }
+  }
   constructor(
     private iscsiBackstorePipe: IscsiBackstorePipe,
     private booleanTextPipe: BooleanTextPipe
   ) {}
 
+  onNodeSelected(tree: TreeModel, node: TreeNode) {
+    TREE_ACTIONS.ACTIVATE(tree, node, true);
+    if (node.data.cdId) {
+      this.title = node.data.name;
+      const tempData = this.metadata[node.data.cdId] || {};
+
```

Comment:
Why this? Some method name reordering? It'd be great to know which rule is enforcing this...

The main risk with a linter or a static checker is that it may produce lots of changes that don't necessarily bring value (increased readability, code quality, etc), but cause a lot of code shuffling. And code shuffling will be a nightmare the next time we find a bug, we fix it in master and we realize that backporting that to a previous branch will require a lot of manual conflict solving because codebases diverged.

My suggestion would be to replace tslint with eslint disabling these cosmetic rules and ensuring that the amount of changes introduced is minimal (e.g.: 400 files vs. just 10).

From the [eslint rules](https://eslint.org/docs/rules/), I see there are: "possible problems", "suggestions" and "layout". Probably we can start focusing on the 1st category, and checking what changes that outputs. Additionally, I'd compare the other categories with the equivalent ones defined by [tslint](https://palantir.github.io/tslint/rules/), and ensure equivalent rules are enabled. If you needed it you can create a doc/etherpad/etc., where you list the tslint vs eslint rules, and we can help you there if needed.